org.texmacs.TeXmacs.yml

```diff
@@ -0,0 +1,116 @@
+app-id: org.texmacs.TeXmacs
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: texmacs
+rename-desktop-file: texmacs.desktop
+rename-icon: texmacs
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --persist=.TeXmacs
+  - --env=PATH=/app/texlive/bin/:/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin
```

Comment:
You should set this through a wrapper.

Reply:
Add a shell wrapper

```
export PATH=/app/texlive/bin:/app/texlive/bin/$(gcc --print-multiarch):${PATH}

exec command_here "$@"
```

instead of setting it statically here

Same with perl. use `perl -e'print substr($^V, 1)'` to get the version.

Reply:
I understand now. I thought doing this kind of thing was frowned upon.

Reply:
The other solution would be to symlink everything from `/app/texlive/bin/x86_64-linux` to `/app/bin`.

Reply:
> The other solution would be to symlink everything from `/app/texlive/bin/x86_64-linux` to `/app/bin`.

this is not how it's used elsewhere - due to how the SDK is built. The wrapper script is a good way.